ecommerce/models.py

```diff
@@ -192,6 +192,12 @@ def start_date(self):
         else:
             raise ValueError(f"Unexpected product {content_object}")
 
+    @property
+    def price(self):
+        """Return the price"""
+        if self.latest_version:
+            return self.latest_version.price
+
```

Comment:
I didn't get why we separated it out from the admin.py file but, Are we using this property?

Reply:
Separated to use it in the Wagtail ViewSets and the Django admin.